app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -356,6 +360,16 @@ public void onCreate(Bundle savedInstanceState) {
         floatingBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+<<<<<<< HEAD
```

Comment:
This looks like it's left over from a merge or rebase gone bad. These markers should never have ended up in a commit. Please check what happened and amend the commit so these aren't in here.